Source/WebKit/WebProcess/WebCoreSupport/WebChromeClient.cpp

```diff
@@ -730,20 +730,20 @@ OptionSet<PointerCharacteristics> WebChromeClient::pointerCharacteristicsOfAllAv
 
 #if ENABLE(POINTER_LOCK)
 
-bool WebChromeClient::requestPointerLock()
+bool WebChromeClient::requestPointerLock(CompletionHandler<void(bool)>&& completion)
 {
     RefPtr page = m_page.get();
     if (!page)
         return false;
```

Comment:
We need to call the CompletionHandler if there's no page.  Also, we may be able to remove the return value of this function.